ibm/mas_devops/roles/suite_install/tasks/airgap-catalogs.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
- Anything airgap related should go into the mas_airgap extension that is being worked on.
- Catalog installation is performed in ocp_setup_mas_deps role already

Reply:
@durera - Thanks for the review 
Have removed the airgap extension from suite_install